lib/pharos/phases/migrate_worker.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Phases
+    class MigrateWorker < Pharos::Phase
+      def call
+        migrate_0_5_to_0_6 if migrate_0_5_to_0_6?
+      end
+
+      def migrate_0_5_to_0_6?
+        !@ssh.file('/etc/kubernetes/kubelet.conf').read.include?('localhost:6443')
```

Comment:
The `file.read(...)` will fail if the file does not yet exist, causing new installs to fail:

```
==> Pharos::Phases::MigrateWorker @ 167.99.36.158 167.99.36.223
    $ sudo cat /etc/kubernetes/kubelet.conf
    $ sudo cat /etc/kubernetes/kubelet.conf
    cat:     /etc/kubernetes/kubelet.conf    : No such file or directory    
    cat:     ! 1
    /etc/kubernetes/kubelet.conf    : No such file or directory [Pharos::Phases::MigrateWorker @ 167.99.36.158] Pharos::SSH::RemoteCommand::ExecError: SSH exec failed with code 1: sudo cat /etc/kubernetes/kubelet.conf
cat: /etc/kubernetes/kubelet.conf: No such file or directory
    
SSH exec failed with code 1: sudo cat /etc/kubernetes/kubelet.conf
cat: /etc/kubernetes/kubelet.conf: No such file or directory
/app/lib/pharos/ssh/remote_command.rb:66:in `run!'
/app/lib/pharos/ssh/client.rb:61:in `exec!'
/app/lib/pharos/ssh/remote_file.rb:46:in `read'
/app/lib/pharos/phases/migrate_worker.rb:11:in `migrate_0_5_to_0_6?'
/app/lib/pharos/phases/migrate_worker.rb:7:in `call'
/app/lib/pharos/phase_manager.rb:70:in `block in apply'
/app/lib/pharos/phase_manager.rb:26:in `block (2 levels) in run_parallel'
```